controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -235,37 +210,42 @@ def test_delete_volume_succeeds(self):
     def test_get_snapshot_return_correct_value(self):
         snapshot_name = "snapshot"
         snapshot_volume_name = "snapshot_volume"
+        snapshot_volume_wwn = "123456789"
         xcli_snapshot = self._get_single_snapshot_result_mock(snapshot_name, snapshot_volume_name)
         self.mediator.client.cmd.vol_list.return_value = xcli_snapshot
-        res = self.mediator.get_snapshot(snapshot_name)
+        res = self.mediator.get_snapshot(snapshot_volume_wwn, snapshot_name)
         self.assertTrue(res.name == snapshot_name)
-        self.assertTrue(res.volume_name == snapshot_volume_name)
+        self.assertTrue(res.source_volume_id == snapshot_volume_wwn)
```

Comment:
In cases of comparing two elements, we prefer to use `self.assertEqual` instead because it prints a meaningful error (in contrast to `self.assertTrue`).
I suggest to change it to all relevant cases.

Reply:
done